docs/configuration/images.md

```diff
@@ -16,4 +16,70 @@ FIXME: add content
+
+![Manipulate LLNL Template Position](img/manipulate_template.gif)
+
+Depending on the instrument you are working on you will follow one of two patterns for the remaining detectors:
+
+1. **Single Image**: Instruments like PXRDIP and BBXRD have a single image with multiple detectors that will be cropped. In cases like these you can simply select a new detector from the drop-down menu and that template will be applied. As each boundary is accepted the dashed lines will remain so that it is clear what has already been cropped.
+2. **Multi-Image**: Instruments like TARDIS have an image per detector. For these cases the next detector should be chosen before the image is loaded (although it is not neccessary, that can still be changed after) and the new template will not be applied until that new image is loaded.
+
+For either path toggling the selected detector before `Save detector boundary` has been selected will change the current template.
+
+*Note*: There is one special case at the moment - Image Plate 3 for TARDIS. The full boundary of the plate is provided to make it easier to align, but after it is saved you will see the boundary shrink. This is to further crop off useless data at the edges.
+
+![TARDIS IP3 Crop](img/TARDIS_IP3.gif)
+
+### Edit Detector Outline
+
+Depending on the image or prefered color map the default settings for the template may be hard to see, or you may want to differentiate between multiple detectors with different boundaries. For these needs there is the option to change the color, line style and thickness.
+
+![Template Style Options](img/template_style.gif)
+
+For the PXRDIP instrument the bounding box inputs will also be enabled. There is sometimes a need to edit the exact size of these templates and these inputs allow for that. At the moment this only applies to PXRDIP and only allows the boundaries to be made smaller than the defaults.
+
+### Finalize Instrument
+
+Each time that a template is saved its name is added to the `Finalize Instrument` section and the option to `Complete and Reload Instrument` is enabled.
+
+![Finalize LLNL Instrument Information](img/finalize_llnl.png)
+
+At any point in the workflow you can use the `Cancel LLNL Import` to clear the progress you've made and return everything to its defaults. This is the only way to re-enable instrument selection once the first image has been loaded.
+
+If everything seems correct you can select `Complete and Reload Instrument`. This will crop all of the images as well as build a new config file with updated row and column sizes as well as tilt values (if appropriate). The config that you selected at the very beginning will be used for this step and will just be updated internally where neccessary. This new configuration file as well as the newly cropped images will all be loaded into the GUI.
```

Comment:
```suggestion
If everything seems correct you can select `Complete and Reload Instrument`. This will crop all of the images as well as build a new configuration with updated row and column sizes as well as tilt values (if appropriate). The configuration that you selected at the very beginning will be used for this step and will just be updated internally where necessary. This new configuration as well as the newly cropped images will all be loaded into the GUI.
```